courses/free-courses-en.md

```diff
@@ -334,6 +334,7 @@
 
 #### SQL
 
+* [Intro to SQL by Khan Academy](https://www.khanacademy.org/computing/computer-programming/sql)
```

Comment:
Already listed in interactive-tutorials: https://github.com/EbookFoundation/free-programming-books/blob/9c6474dcb51a5d8c86abd2d51b344a6a7c6abbc6/more/free-programming-interactive-tutorials-en.md?plain=1#L385

Removing...
```suggestion
```